app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -24,6 +24,12 @@
   </div>
 </div>
 
+<div class="govuk-grid-row app-print-link">
```

Comment:
Should we also had the class `app-!-print-display-none`?

Reply:
@fofr https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1607/commits/a33fd8136e20ff7c74d37204738db6b4832dd94a